infrastructure/scripts/lib/k8s.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+function wait-for-service {
+    local pod=$1
+    local service=$2
+    local port=$3
+    local delay=$4
+    local name=$5
+
+    kubectl exec ${pod} -- /bin/sh -c "while ! nc -z ${service} ${port} >/dev/null; do sleep ${delay}; echo Waiting for ${name} to start...; done"
+}
+
+function wait-for-running-pod {
+    local name=$1
+
+    while ! `kubectl get pod --field-selector="metadata.name=${name},status.phase=Running" 2>/dev/null| grep -q ${name}`
+    do
+        echo "Waiting for Airy startup helper to start..."
+        sleep 5
+    done
+}
+
+function wait-for-ingress-service {
+    while ! `kubectl -n kube-system get service --field-selector="metadata.name=traefik" 2>/dev/null | grep -q "traefik"`
+    do
+        echo "Waiting for ingress to start... "
+        sleep 5
+    done
+}
+
+function wait-for-sa {
```

Comment:
`wait-for-service-account` would follow the convention of the rest of the file and make it much clearer